routingpy/convert.py

```diff
@@ -108,3 +113,35 @@ def seconds_to_iso8601(seconds):
         iso8601_duration += f"{seconds}S"
 
     return iso8601_duration
+
+
+def timestamp_to_tz_datetime(timestamp: int, timezone: str) -> datetime.datetime:
```

Comment:
I basically took out the time converting functions and put them here. I also consolidated them a bit more.